elements/classpower.lua

```diff
@@ -115,7 +148,7 @@ local function Update(self, event, unit, powerType)
 	* event         - the event that triggered the update
 	--]]
 	if(element.PostUpdate) then
-		return element:PostUpdate(cur, max, oldMax ~= max, powerType, event)
+		return element:PostUpdate(full, partial, max, oldMax ~= max, powerType, event)
```

Comment:
Not related to the commit itself but `event` should be the first argument for consistency reasons, shouldn't it?